.circleci/docker-compose.yml

```diff
@@ -14,4 +14,4 @@ reaction:
 
 mongo:
   image: mongo:latest
-  command: mongod --storageEngine=wiredTiger
+  command: mongod --storageEngine=wiredTiger --bind_ip_all
```

Comment:
What's the story here?

Reply:
https://jira.mongodb.org/browse/SERVER-28229 -- MongoDB as of 3.5.7 now binds to localhost-only (see also mongodb/mongo@60636b4).

There is a new --bind_ip_all flag introduced to resolve.